app/src/main/java/com/example/makeitso/common/composable/PermissionDialogComposable.kt

```diff
@@ -0,0 +1,48 @@
+package com.example.makeitso.common.composable
+
+import androidx.compose.material.AlertDialog
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.res.stringResource
+import com.example.makeitso.R.string as AppText
+
+@Composable
+fun PermissionDialog(onRequestPermission: () -> Unit) {
+    var showWarningDialog by remember { mutableStateOf(true) }
+
+    if (showWarningDialog) {
+        AlertDialog(
+            title = { Text(stringResource(id = AppText.notification_permission_title)) },
+            text = { Text(stringResource(id = AppText.notification_permission_description)) },
+            confirmButton = {
```

Comment:
Shouldn you have a dismissButton?

Reply:
Not in this case, the only action available is to launch the system dialog to ask for the permission. Then, on the system dialog the user has the option to either enable the permission, or deny it. But this system dialog must be launched anyway.